components/DeleteModal.js

```diff
@@ -0,0 +1,59 @@
+import { Button, Overlay } from "./ReusedStyles";
+import styled from "styled-components";
+
+export default function DeleteModal({
+  nestingbox,
+  deleteCard,
+  setShowDeletePopup,
+  setToEditCardID,
+}) {
+  return (
+    <Overlay
+      onClick={(event) => {
+        event.stopPropagation();
+        setShowDeletePopup(null);
+      }}
+    >
+      <Modal onClick={(event) => event.stopPropagation()}>
+        <ModalP isPrimary>Are you sure you want to delete the data?</ModalP>
```

Comment:
Just a personal preference: a "better" naming for this styled-component

```suggestion
        <ModalText isPrimary>Are you sure you want to delete the data?</ModalText>
```